backend/models/User.js

```diff
@@ -2,32 +2,28 @@ import mongoose from 'mongoose';
 
 const UserSchema = mongoose.Schema(
     {
-        firstName: {
-            type: String, 
-            required: true
-        },
-        lastName: {
-            type: String, 
-            required: true
-        },
-        age: {
-            type: Number, 
-            required: true
-        }, 
-        username: {
-            type: String,
-            required: true,
-            unique: true
-        },
-        password: {
-            type: String,
-            required: true
-        },
-        description: {
-            type: String,
-            required: true
-        },
+        email: {type: String, required: true},
+        username: {type: String, required: true, unique: true, trim: true},
+        password: {type: String, required: true},
+        firstName: {type: String, required: true, trim: true},
```

Comment:
I think we also need a way to store the token right?

Reply:
yeah that's the "sub"